modules/jooby-assets/src/main/java/org/jooby/assets/Assets.java

```diff
@@ -516,9 +516,8 @@ public void configure(final Env env, final Config config, final Binder binder) t
     // live compiler?
     AssetHandler handler;
     if (watch) {
-      Path publicDir = Paths.get(conf.getString("user.dir"), "public");
       Path workdir = Paths.get(conf.getString("assets.outputDir"));
-      handler = new FileSystemAssetHandler("/", workdir, publicDir);
+      handler = new FileSystemAssetHandler("/", workdir);
```

Comment:
you need to revert this change. We do need multi sources for file system asset handler

Reply:
Hi Edgar, this change was coming from f10ac3f87 (upstream/master merge. I surely will revert it, sorry!